frontend/components/analytics/Matomo.tsx

```diff
@@ -1,27 +1,43 @@
 import { MATOMO_URL } from '@config/env';
-import Head from 'next/head';
+import { useRouter } from 'next/router';
+import * as React from 'react';
+import Script from 'next/script';
+
+declare const _paq: string[][] | undefined;
 
 export function Matomo() {
+   const router = useRouter();
+   React.useEffect(() => {
+      const handleRouteChange = (url: string) => {
+         if (typeof window !== 'undefined' && _paq) {
+            _paq.push(['setCustomUrl', url]);
+            _paq.push(['setDocumentTitle', document.title]);
+            _paq.push(['trackPageView']);
```

Comment:
Hey I think we should look at doing something similar to: https://github.com/iFixit/ifixit/commit/7fa161dee14bd23df385f4129994f39627955ee3

Just adding some types and nice function calls around Matomo so we never need to worry about `_paq` and how it works

Reply:
Good idea!